src/server/scripts/Outland/HellfireCitadel/BloodFurnace/instance_blood_furnace.cpp

```diff
@@ -39,6 +39,16 @@ ObjectData const creatureData[] =
     { NPC_KELIDAN, DATA_KELIDAN }
 };
 
+Creature *GetBroggok(InstanceScript *instance)
```

Comment:
Functions in C++ added outside the class scope are added as global symbols, meaning this will be available universally in the entire project

Reply:
Do you suggest making the change inline in both places where the function is invoked?